universal/src/crypto/public_key.cpp

```diff
@@ -81,6 +91,12 @@ int CurveStringToNid(const std::string_view& curve_str) {
   return it->second;
 }
 
+#ifdef USERVER_FEATURE_WOLFSSL
+#ifndef EC_KEY_set_public_key_affine_coordinates
+#warning "EC_KEY_set_public_key_affine_coordinates does not provided by wolfSSL"
```

Comment:
is it an old wolfssl version limitation?

Reply:
At the point of PR, 5.7.0 is the latest stable version.
WolfSSL does not provide function with suffix `_coordinates`, but `EC_KEY_set_public_key`
See https://github.com/wolfSSL/wolfssl/blob/master/wolfssl/openssl/ec.h